src/sync.ts

```diff
@@ -59,12 +59,12 @@ export class Sync {
                 tokenAvailable &&
                 gistAvailable
               ) {
-                return globalCommonService.autoUploadService.StartWatching();
+                return globalCommonService.HandleStartWatching();
```

Comment:
`HandleStartWatching` and `StopWatching` are `async` function, can you use `await` before calling them everywhere.

Reply:
Sure